src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int COUNT = 5;
+    private static final int VALUE = 9;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figures getRandomFigure() {
+        int randomFigureNumber = random.nextInt(COUNT);
+        switch (randomFigureNumber) {
+            case 0 :
+                Square square = new Square();
+                return getRandomSquare(square);
+            case 1 :
+                Circle circle = new Circle();
+                return getRandomCircle(circle);
```

Comment:
```suggestion
            case 1 :
                return getRandomCircle();
```
створи краще об'єкт всередині методу вже